test/python/pulse/test_schedule.py

```diff
@@ -348,6 +348,48 @@ def my_test_par_sched_two(x, y, z):
 
         self.assertEqual(par_sched.parameters, ('x', 'y', 'z'))
 
+    def test_schedule_with_acquire_on_single_and_multiple_qubits(self):
+        """Test schedule with acquire on single and multiple qubits."""
+        acquire = Acquire(10)
+
+        # schedule with acquire on single qubit
+        sched_single = Schedule()
+        for i in range(self.config.n_qubits):
+            sched_single = sched_single.insert(10, acquire(self.config.acquire(i),
+                                                           MemorySlot(i),
+                                                           RegisterSlot(i)))
+
+        self.assertEqual(len(sched_single.instructions), 2)
+        self.assertEqual(len(sched_single.channels), 6)
+
+        # schedule with acquire on multiple qubits
```

Comment:
Do you think this could be broken up into multiple tests?

Reply:
done :)